src/registrar/templates/domain_detail.html

```diff
@@ -5,6 +5,28 @@
   {{ block.super }}
   <div class="margin-top-4 tablet:grid-col-10">
 
+    <div
+        class="usa-summary-box dotgov-status-box margin-top-3 padding-left-2{% if domain.state == 'unknown' or domain.state == 'dns needed'%} dotgov-status-box--action-need{% endif %}" 
```

Comment:
```suggestion
        class="usa-summary-box dotgov-status-box margin-top-3 padding-left-2{% if domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED%} dotgov-status-box--action-need{% endif %}" 
```
(Optional) Not sure if it'll let you from this context (it should) but I would recommend replacing these guys with the above